python_src/src/lamp_py/performance_manager/l0_rt_trip_updates.py

```diff
@@ -122,13 +65,27 @@ def get_and_unwrap_tu_dataframe(
     process_logger = ProcessLogger("tu.get_and_unwrap_dataframe")
     process_logger.log_start()
 
-    events = pandas.Series(dtype="object")
+    trip_updates = pandas.DataFrame()
+
+    rename_mapper = {
+        "trip_update.timestamp": "timestamp",
+        "trip_update.stop_time_update.stop_id": "stop_id",
+        "trip_update.stop_time_update.arrival.time": "tu_stop_timestamp",
+        "trip_update.trip.direction_id": "direction_id",
+        "trip_update.trip.route_id": "route_id",
+        "trip_update.trip.start_date": "start_date",
+        "trip_update.trip.start_time": "start_time",
+        "trip_update.vehicle.id": "vehicle_id",
+        "trip_update.trip.trip_id": "trip_id",
+    }
```

Comment:
rename columns to previously used values as soon as possible to avoid changing entire pipeline.